index.html

```diff
@@ -8,16 +8,16 @@
   <link rel="stylesheet" href="style.css">
   <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
 </head>
-<body>
   <header>
     <a href="#"><h2>John Doe</h2></a>
     <nav>
-      <img src="menu.png" class="menu" alt="">
-      <ul class="menu_items">
-        <li>Portfolio</li>
-        <li>About</li>
-        <li>Contact</li>
-        <li><img src="mail.png" class="mail" alt="image"></li>
+      <button type="button" id="toggleBtn"><img src="/images/menu.png" class="menu" alt=""></button>
+      <ul class="menu_items" id="menuList">
+        <li><button type="button" id="btn_close">X</button></li>
+        <li><a href="#project-s">Portfolio</a><span>></span></li>
+        <li><a href="#headline">About</a><span>></span></li>
+        <li><a href="#contact-me">Contact</a><span>></span></li>
+        <li><img src="/images/mail.png" class="mail" alt="image"><span>></span></li>
       </ul>
```

Comment:
- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that the _hamburger_ and the _close button_ are behaving as buttons when the mouse hovers on them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _hamburger_ and the _close button_ on _mouse hover_ events. We can additionally change the color or opacity of these objects during the hover event.